frontend/packages/dev-console/src/components/import/git/GitTypeSelector.tsx

```diff
@@ -7,6 +7,7 @@ import {
   GitAltIcon,
 } from '@patternfly/react-icons/dist/esm/icons';
 import { FormikValues, useFormikContext } from 'formik';
+import get from 'lodash-es/get';
```

Comment:
```suggestion
import * as _ from 'lodash';
```
Make this the first import like done in other files

Reply:
Updated the PR. PTAL.